### DISCUSSION TITLE
Community Meeting #1

### DISCUSSION THREAD

**Discussion by markcheret** [markcheret/footnotes#112] [Q&A]
- publish slack URL?
- upcoming release
- add additional developer resources
- add additional support reps

**Comment by Rumperuu** [comment]
I've just started a thread about the next release + 1 ([2.8](https://github.com/markcheret/footnotes/discussions/116)) 

If we haven't got a date in mind for this yet, I think it might be a good idea to coincide the first discussion with the bigger 2.8 release — we can put a note along with the 2.7.1 release to give people a heads-up now, and then hit the ground running with a major revamp _and_ new community engagement simultaneously.

**Reply by markcheret** [reply]
agreed on all points, only caveat... I'm thinking of kicking community meetings off before 2.8 drops as a warm up exercise and to get some housekeeping items out of the way :)

**Reply by Rumperuu** [reply]
Fair enough

**Comment by Rumperuu** [comment]
Additional agenda:

- agree CoC wording (from https://github.com/markcheret/footnotes/pull/123#issuecomment-822052130)
- rewrite project description (from https://github.com/markcheret/footnotes/pull/120#issuecomment-822023706)

**Comment by markcheret** [comment]
add "voices from the support forum" and ratings

**Comment by markcheret** [comment]
The meeting is public for everyone who wants to join:
https://doodle.com/poll/cntgwp6fytviu4aq

**Comment by markcheret** [comment]
Resources to add to the project:
- Staging website for project access and manual testing
- Beta testers (actual people)
- automated code testing
- automated UI/UX testing

**Comment by markcheret** [comment]
Post-event writeup provided here: https://cheret.tech/2021/04/first-footnotes-community-meeting/
